packages/bits/spec/components/dialog/dialog.e2e.ts

```diff
@@ -28,6 +28,8 @@ describe("USERCONTROL Dialog", () => {
     let cancelButton: ElementFinder;
     let actionButton: ElementFinder;
     let themeSwitcher: WebElement;
+    const setLocation = (url: string) =>
+        browser.executeScript((pUrl: string) => window.location.href = `/#/${pUrl}`, url);
```

Comment:
`browser.executeScript()` returns a Promise, so the function itself must be async. Later in the code, when calling the function, we'll need to await it to properly resolve the returned promise.

Reply:
@dmytrofirsov, do you think this would be a handy function to put in the Helpers class along with the existing `prepareBrowser` function? It's just an idea and not really necessary for this PR.

Reply:
Makes sense @Blakesalot . @dmytrognatyshyn  please add this function to the helpers class, make it static and 
 describe what the function does in a comment

Reply:
@dmytrofirsov ok